<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Saving and Loading</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QComboBox" name="layerDataCombo">
            <item>
             <property name="text">
              <string>XML</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Base64 (uncompressed)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Base64 (gzip compressed)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Base64 (zlib compressed)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CSV</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="reloadTilesetImages">
            <property name="text">
             <string>&amp;Reload tileset images when they change</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="enableDtd">
            <property name="toolTip">
             <string>Not enabled by default since a reference to an external DTD is known to cause problems with some XML parsers.</string>
            </property>
            <property name="text">
             <string>Include &amp;DTD reference in saved maps</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Store tile layer data as:</string>
            </property>
            <property name="buddy">
             <cstring>layerDataCombo</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Interface</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&amp;Language:</string>
              </property>
              <property name="buddy">
               <cstring>languageCombo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="languageCombo"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="2">
             <widget class="QToolButton" name="bgColorReset">
              <property name="text">
               <string>Default</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Grid color:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="Tiled::Internal::ColorButton" name="bgColor"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Background color:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="Tiled::Internal::ColorButton" name="gridColor"/>
            </item>
            <item row="2" column="3">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="gridColorReset">
              <property name="text">
               <string>Default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
          <item>
           <widget class="QCheckBox" name="openGL">
            <property name="text">
             <string>Hardware &amp;accelerated drawing (OpenGL)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>381</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Object Types</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Object Types</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableView" name="objectTypesTable">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QToolButton" name="addObjectTypeButton">
              <property name="toolTip">
               <string>Add Object Type</string>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="tiled.qrc">
                <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removeObjectTypeButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Remove Selected Object Types</string>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset resource="tiled.qrc">
                <normaloff>:/images/22x22/remove.png</normaloff>:/images/22x22/remove.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="importObjectTypesButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Import...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportObjectTypesButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Export...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Automapping</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="autoMapWhileDrawing">
         <property name="text">
          <string>Use Automapping, when drawing into layers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Zomboid</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="configBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Config Directory</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>This is where config files are stored.  Change it with the 'config' program.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="configLayout">
            <item>
             <widget class="QLineEdit" name="configDirectory">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="configBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>WorldEd Project Files</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Used for adjacent maps and lots.  Restart TileZed to see changes.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listPZW">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>96</height>
             </size>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QToolButton" name="addPZW">
              <property name="toolTip">
               <string>Add Project</string>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="tiled.qrc">
                <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removePZW">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Remove Selected Project</string>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset resource="tiled.qrc">
                <normaloff>:/images/22x22/remove.png</normaloff>:/images/22x22/remove.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="raisePZW">
              <property name="toolTip">
               <string>Raise Project</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="tiled.qrc">
                <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonIconOnly</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="lowerPZW">
              <property name="toolTip">
               <string>Lower Project</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="tiled.qrc">
                <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonIconOnly</enum>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showAdjacent">
         <property name="text">
          <string>Show adjacent maps.  Requires closing/opening a map to see changes.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::ColorButton</class>
   <extends>QToolButton</extends>
   <header location="global">colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>layerDataCombo</tabstop>
  <tabstop>enableDtd</tabstop>
  <tabstop>reloadTilesetImages</tabstop>
  <tabstop>openGL</tabstop>
  <tabstop>objectTypesTable</tabstop>
  <tabstop>addObjectTypeButton</tabstop>
  <tabstop>removeObjectTypeButton</tabstop>
  <tabstop>importObjectTypesButton</tabstop>
  <tabstop>exportObjectTypesButton</tabstop>
  <tabstop>autoMapWhileDrawing</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
